kolibri/plugins/learn/assets/src/constants.js

```diff
@@ -1,11 +1,10 @@
 // a name for every URL pattern
 const PageNames = {
-  TOPICS_ROOT: 'TOPICS_ROOT',
+  CHANNELS: 'CHANNELS',
   TOPICS_CHANNEL: 'TOPICS_CHANNEL',
   TOPICS_TOPIC: 'TOPICS_TOPIC',
   TOPICS_CONTENT: 'TOPICS_CONTENT',
-  LEARN_ROOT: 'LEARN_ROOT',
-  LEARN_CHANNEL: 'LEARN_CHANNEL',
+  LEARN_RECOMMENDED: 'LEARN_RECOMMENDED',
   RECOMMENDED_POPULAR: 'RECOMMENDED_POPULAR',
   RECOMMENDED_RESUME: 'RECOMMENDED_RESUME',
   RECOMMENDED_NEXT_STEPS: 'RECOMMENDED_NEXT_STEPS',
```

Comment:
`LEARN_CONTENT` -> `RECOMMENDED_CONTENT`
`SEARCH_ROOT` is no longer used.
`EXAM_ROOT`?

Reply:
EXAM_ROOT was removed in error, reinstating.